src/components/Components/CandidateSection/CandidateSection.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import "./CandidateSection.scss"
+import ViewMoreIcon from "./../../assets/viewMoreIcon.png"
+
+function CandidateSection() {
+  return (
+    <div className='CandidateSection'>
+        <div className='CandidateSection__wrapper'>
+            <h2 className='CandidateSection__wrapper__title'>
+                Meet Candidates
+                <div className='CandidateSection__wrapper__title__rec'></div>
+            </h2>
+            <div className='CandidateSection__wrapper__viewmore CandidateSection__wrapper__viewmore--top'>
```

Comment:
@jasper-prgh Jasper, what is the benefit of creating 2 elements for viewmore--bottom and displaying them depending on the screen size? I mean, isn't it cleaner just to change the position of viewmore--bottom to absolute, depending on the screen size (for mobile) and move it to the bottom?

here what I mean https://vdb99b.csb.app/
https://codesandbox.io/s/meet-candidates-vdb99b?file=/src/styles.css

Reply:
@tinkabel85 Your solution works just as well. However I find the solution of having two elements easier to understand - since the positioning may get difficult if you dont know how big the content (cards in this case) is going to be

Reply:
ok, then we have to 'play' with parents' element position (static/relative) I guess, depending on the screen size.
@RavLiya we can also remove the words "top" and "down" in the text "View More top"/ "View More  bottom", since classes indicates their position

Reply:
Thanks for pointing that out. silly mistake!